gen/vim_wsdl.rb

```diff
@@ -866,7 +866,7 @@ def peek
   def self.header(name)
     return <<EOF
 /*
-Copyright (c) 2014-2018 VMware, Inc. All Rights Reserved.
+Copyright (c) #{ENV['COPYRIGHT_DATE_RANGE'] || '2014-2018'} VMware, Inc. All Rights Reserved.
```

Comment:
Probably fine just to have 2014-2021 for all, then again guess we haven't updated this in a few years 😅

Reply:
Yeah, except I did not update the other generated files. I am planning on following up with another PR that updates all of the generated files to 7.0 U2 (7.0.2), at which point they will all use `2014-2021`. The code is to make sure it is possible to accurately reflect the updated copyright header for updates to specific bindings but not others.